pipelines/annotations.snakefile

```diff
@@ -5,7 +5,7 @@ import re
 import yaml
 
 
-configfile: "config/deeprvat_annotation_config.yaml"
+configfile: "../example/config/deeprvat_annotation_config.yaml"
```

Comment:
I guess we shouldn't point to the example directory for the config file, right? It should be one provided by the user. That is, I wouldn't change this line.